backend/file-system/src/main/java/com/htec/filesystem/controller/FileController.java

```diff
@@ -58,4 +60,11 @@ public ResponseEntity uploadFile(@RequestBody Map<String, Pair<String, String>>
 
         return ResponseEntity.status(HttpStatus.OK).body(new TextResponseMessage("File Uploaded", HttpStatus.OK.value()));
     }
+
+    @PutMapping("/move-to-trash")
```

Comment:
read about PATCH method and what is its purpose

Reply:
Currently we have some problems with Cors when we are using patch method